### ISSUE TITLE
cf-deployment use undefined keys from capi-release specs

### ISSUE THREAD

**Issue by romain-dartigues** [cloudfoundry/cf-deployment#955]
### What is this issue about?

We noticed the `cloud_controller_ng` and `cloud_controller_worker` in the `cf-deployment.yml` refer to some keys which are undefined in the [`capi-release`](https://github.com/cloudfoundry/capi-release/) specs (we first found it on the generated manifest at `/instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/bulk_api_password`).

<details>
<summary>A simple Python script to check</summary>
Expect to be run in a directory containing both `cf-deployment` and `capi-release` cloned repositories.

```python
#!/usr/bin/env python3

import yaml

cf_deployment = yaml.safe_load(open('cf-deployment/cf-deployment.yml'))

for ig in cf_deployment['instance_groups']:
    for job in ig.get('jobs', {}):
        if job.get('name') == 'cloud_controller_ng':
            cloud_controller_ng = job
        elif job.get('name') == 'cloud_controller_worker':
            cloud_controller_worker = job

cloud_controller_ng_spec = yaml.safe_load(open('capi-release/jobs/cloud_controller_ng/spec'))['properties']
cloud_controller_worker_spec = yaml.safe_load(open('capi-release/jobs/cloud_controller_worker/spec'))['properties']

print('cloud_controller_ng:')
for missing_from_spec in sorted(set(cloud_controller_ng).difference(cloud_controller_ng_spec)):
    print(f'\t{missing_from_spec!s}: {cloud_controller_ng[missing_from_spec]!r}')

print()

print('cloud_controller_worker:')
for missing_from_spec in sorted(set(cloud_controller_worker).difference(cloud_controller_worker_spec)):
    print(f'\t{missing_from_spec!s}: {cloud_controller_worker[missing_from_spec]!r}')
```
</details>

### What version of [cf-deployment](https://github.com/cloudfoundry/cf-deployment/releases) are you using?

We are currently using the v17.1.0.

### Please include the `bosh deploy...` command, including all the operations files (plus any experimental operation files you're using):

n/a

### What IaaS is this issue occurring on?

CloudStack and vSphere.

### Is there anything else unique or special about your setup?

No relevant.

### Tag your pair, your PM, and/or team!

@psycofdj

**Comment by jochenehret** [comment]
The obsolete bulk_api_password is already addressed in https://github.com/cloudfoundry/cf-deployment/pull/956

**Comment by jochenehret** [comment]
This is resolved in [v19.0.0.](https://github.com/cloudfoundry/cf-deployment/releases/tag/v19.0.0).
